src/script.cc

```diff
@@ -18,7 +18,7 @@
 
 namespace amber {
 
-Script::Script() = default;
+Script::Script() : virtual_files_(new VirtualFileStore()) {}
```

Comment:
nit: `virtual_files_(std::make_unique<VirtualFileStore>())`

Reply:
`make_unique` was introduced in C++14. I thought amber targeted 11?

Reply:
Been working in tint too long, `virtual_files_(MakeUnique<VirtualFileStore>())`

Reply:
Ack.

Reply:
Done.